packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -16,6 +16,15 @@
 #import "FLTImagePickerPhotoAssetUtil.h"
 #import "FLTPHPickerSaveImageToPathOperation.h"
 
+/**
+ * Returns the value for the given key in 'dict', or nil if the value is
+ * NSNull.
+ */
+NSObject *GetNullableValueForKey(NSDictionary *dict, NSString *key) {
+  NSObject *value = dict[key];
+  return value == [NSNull null] ? nil : value;
+}
```

Comment:
Actually the analyzer doesn't like that this is a `NSObject`.
```
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:409:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *maxWidth = GetNullableValueForKey(self->_arguments, @"maxWidth");
              ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:410:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *maxHeight = GetNullableValueForKey(self->_arguments, @"maxHeight");
              ^           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:411:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *imageQuality = GetNullableValueForKey(self->_arguments, @"imageQuality");
              ^              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:492:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *maxWidth = GetNullableValueForKey(_arguments, @"maxWidth");
              ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:493:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *maxHeight = GetNullableValueForKey(_arguments, @"maxHeight");
              ^           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/folders/tn/f_9sf1xx5t14qm_6f83q3b840000gn/T/cirrus-ci-build/packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m:494:15: error: incompatible pointer types initializing 'NSNumber *' with an expression of type 'NSObject *' [-Werror,-Wincompatible-pointer-types]
    NSNumber *imageQuality = GetNullableValueForKey(_arguments, @"imageQuality");
              ^              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
```
```suggestion
id GetNullableValueForKey(NSDictionary *dict, NSString *key) {
  id value = dict[key];
  return value == [NSNull null] ? nil : value;
}
```

Reply:
(woo analyzer running in presubmit!)

Reply:
I thought that might happen but couldn't remember how NSObject* was handled. I meant to check locally before pushing 😛 

Fixed! Also fixed the "non-nullable" arguments on `scaledImage:...` that had handling for `NSNull` to be nullable now that they are actually nil.